src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1112,13 +1097,19 @@ public OctoTentacle getTentacle()
 
 
     /**
-     * Handles an RTCP packet coming from a specific endpoint.
+     * Handles an RTP/RTCP packet coming from a specific endpoint.
      * @param packetInfo
      */
-    public void handleIncomingRtcp(PacketInfo packetInfo)
+    public void handleIncomingPacket(PacketInfo packetInfo)
     {
         Packet packet = packetInfo.getPacket();
-        if (packet instanceof RtcpFbPliPacket
+        if (packet instanceof RtpPacket)
```

Comment:
why not just add conditions to this if that exclude the tests in the `else if` to avoid the duplicate `else` thing?  or add a helper so we could have something like:

```
if (packet instanceof RtpPacket && notSingleDestRtcp(packetInfo) {
   // fan out logic
} else {
  // Single-dest rtcp logic
}